spec/features/review_card_spec.rb

```diff
@@ -1,8 +1,23 @@
 require 'rails_helper'
 
-feature 'Review Card', %q{'It should
-  be the ability to
-  check the knowledge Card'} do
+feature 'Review Card', 'It should be the ability to check the knowledge Card' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.